virt/virt-2-4-release-notes.adoc

```diff
@@ -100,15 +100,16 @@ from being inadvertently started before an upload has completed.
 
 [id="virt-2-4-known-issues"]
 == Known issues
-//https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
+//This is work in progress.
 
 * KubeMacPool is disabled in {VirtProductName} {VirtVersion}. This means that a secondary
 interface of a Pod or virtual machine obtains a randomly generated MAC address rather
 than a unique one from a pool. Although rare, randomly assigned MAC addresses can conflict.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1816971[*BZ#1816971*])
+//Merge conflict resolution - adding this text back on July 21: //Bug 1795410 - this known issue was open when I checked on July 17th.
+//BZ#1855067.
+* Due to an unknown reason, the memory consumption for the volume type `containerDisk` may increase slowly and exceed the memory limit. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1855067[*BZ#1855067*])
```

Comment:
Instead of saying "Due to an unknown reason," I would say something like "For unknown reasons,"

To bring my comments together, here's a potential rewrite: "For unknown reasons, memory consumption for the `containerDisk` volume type might gradually increase until it exceeds the memory limit. To resolve this issue, restart the VM."

Reply:
Content replaced.